projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/vdk-jupyterlab-extension/handlers.py

```diff
@@ -7,15 +7,28 @@
 from jupyter_server.base.handlers import APIHandler
 from jupyter_server.utils import url_path_join
 
+from .job_data import JobData
 from .vdk_options.vdk_options import VdkOption
 from .vdk_ui import VdkUI
 
 
-class RunJobHandler(APIHandler):
+class JobDataHandler(APIHandler):
```

Comment:
nit: the name of handler may be more meaningful, suggesting what action or event it is handling?